app/grandchallenge/challenges/forms.py

```diff
@@ -28,6 +29,7 @@ class Meta:
             "use_evaluation",
             "contact_email",
         ]
+        widgets = {"use_evaluation": forms.HiddenInput()}
```

Comment:
Rather than using a hidden input I think that it's best to change the default on the model, create a migration, and then remove it from the form.

Reply:
Rest looks great!

Reply:
The default is already True actually, so I can just go ahead and delete the field. :-)